orchestration/dags/data_gcp_dbt/models/analytics/firebase/firebase_home_micro_conversion.sql

```diff
@@ -15,9 +15,9 @@ SELECT
     , COUNT( CASE WHEN booking_timestamp IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings
     , COUNT( CASE WHEN booking_id IS NOT NULL THEN 1 ELSE NULL END) AS nb_bookings_non_cancelled
     , SUM(delta_diversification) AS total_diversification
-FROM `{{ bigquery_analytics_dataset }}.firebase_home_funnel_conversion`
-LEFT JOIN `{{ bigquery_clean_dataset }}.applicative_database_user` USING(user_id)
-LEFT JOIN `{{ bigquery_analytics_dataset }}.diversification_booking`  USING(booking_id)
+FROM {{ ref('firebase_home_funnel_conversion')}}
+LEFT JOIN {{ ref('int_applicative__user') }} USING(user_id)
```

Comment:
```suggestion
LEFT JOIN {{ source('raw','applicative_database_user') }} USING(user_id)
```
Idem ici on peut mettre directement la source